### DISCUSSION TITLE
debounce SSerialReceived?

### DISCUSSION THREAD

**Discussion by s-seidel** [arendst/Tasmota#15934] [Q&A] [answered]
Hi, I am using Tasmota with RDM6300 reader. Using SerialDelimiter 128 I can get clean readings of the ID, but as usual with this kind of reader it will keep sending the ID for as long as the chip is near the reader. Since there is ButtonDebounce and SwitchDebounce, I was wondering if it's possible to "debounce" serial messages as well.

**Comment by arendst** [comment, marked as answer]
No there is no "SerialDebounce"

Why don't you use the native Tasmota RDM6300 driver (define USE_RDM6300)???????

**Reply by s-seidel** [reply]
Okay I can try that. I was trying to avoid having to compile it myself, but that's probably the next logical step.
